src/middleware/authentication.ts

```diff
@@ -50,16 +50,19 @@ export const isSeller = (req: Request, res: Response, next: NextFunction) => {
   if (!req.user || !('role' in req.user) || req.user.role !== 'seller') {
     return sendResponse(
       res,
-      401,
+      403,
       null,
-      'Not authorized! user should be seller'
+      'Forbidden'
```

Comment:
why did you remove authorized then you put forbidden? please only the status was requested